applications/StructuralMechanicsApplication/custom_elements/small_displacement.h

```diff
@@ -43,11 +43,9 @@ namespace Kratos
 ///@name Kratos Classes
 ///@{
 
-/// Total Lagrangian element for 2D and 3D geometries.
-
 /**
- * Implements a total Lagrangian definition for structural analysis.
- * This works for arbitrary geometries in 2D and 3D
+ * @brief Small displacement element for 2D and 3D geometries.
+ * @details Implements a small displacement definition for structural analysis. This works for arbitrary geometries in 2D and 3D
  */
 
 class KRATOS_API(STRUCTURAL_MECHANICS_APPLICATION) SmallDisplacement
```

Comment:
Why is `KRATOS_API(STRUCTURAL_MECHANICS_APPLICATION)` only done for the `SmallDisplacement`?

Reply:
True

Reply:
Done